easybuild/easyconfigs/m/mrsFAST/mrsFAST-2.6.0.4-intel-compiler.patch

```diff
@@ -0,0 +1,17 @@
+diff -ru mrsfast-2.6.0.4.orig/Makefile mrsfast-2.6.0.4/Makefile
```

Comment:
@pescobar please add a line of comment explaining the patch, and add yourself as an author